src/js/post-excerpt/panel.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * WordPress dependencies
+ */
+const { __ } = wp.i18n;
+const { Button, ExternalLink, TextareaControl } = wp.components;
+const { withSelect, withDispatch } = wp.data;
+const { compose } = wp.compose;
+
+/**
+ * Internal dependencies
+ */
+import { handleClick } from '../helpers';
+
+function PostExcerpt( { excerpt, onUpdateExcerpt } ) {
+	const { select } = wp.data;
+	const postId = select( 'core/editor' ).getCurrentPostId();
+	const buttonText = __( 'Generate excerpt', 'classifai' );
+
+	return (
+		<div className="editor-post-excerpt">
+			<TextareaControl
+				__nextHasNoMarginBottom
+				label={ __( 'Write an excerpt (optional)' ) }
+				className="editor-post-excerpt__textarea"
+				onChange={ ( value ) => onUpdateExcerpt( value ) }
+				value={ excerpt }
+			/>
+			<ExternalLink
+				href={ __(
+					'https://wordpress.org/support/article/settings-sidebar/#excerpt'
```

Comment:
@dkotter Does this link need to be translatable?

Reply:
As mentioned in my comment above, this code was copied from Gutenberg. Looking at WordPress Core translations, it looks like they do have language specific URLs they swap in here, see https://translate.wordpress.org/projects/wp/6.1.x/fr/default/?filters%5Bterm%5D=https%3A%2F%2Fwordpress.org%2Fsupport%2Farticle%2Fsettings-sidebar%2F%23excerpt&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc